### PR TITLE
Carousel movement advancing

### PR DESCRIPTION
**Pull request by Sharonsyra**
**What does this PR do?**

- Have the next slide starting from the top
- testimonial section links open on the same page
- Fix unordered list 

**Description of tasks** 

- User scrollTo() to scroll to the top. 
- Remove target blank on testimonial section
- Use disc list type and add padding left.

**How to test the app** 

[link](https://cfafrica-mooc-dev.appspot.com)

**Screenshots**

screen shot 2017-10-09 at 16 29 36

### CODE REVIEW

File: coursebuilder/app.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-application: mycourse  # FIXME: Replace this with your application id
+application: cfafrica-mooc-dev  # FIXME: Replace this with your application id
```
Comment (chegejames) [inline coursebuilder/app.yaml]: @Sharonsyra you may want not to commit this as it's going to change from deployment to deployment.

Comment (Sharonsyra) [reply coursebuilder/app.yaml]: Thanks, I will fix that.

File: coursebuilder/modules/explorer/_static/js/course.js [hunk 2]
```diff
@@ -8,10 +8,13 @@ $(document).ready(function documentReady() {
     margin: 10,
     nav: true,
     navText: [
-      '<i class="material-icons small">chevron_left</i>',
-      '<i class="material-icons small">chevron_right</i>'
+      '<i class="material-icons small" onclick="window.scrollTo(20, 20)">chevron_left</i>',
```
Comment (chegejames) [inline coursebuilder/modules/explorer/_static/js/course.js]: @Sharonsyra this works. However, one can also navigate using the owl-dots hence callbacks may be more robust.

Comment (chegejames) [reply coursebuilder/modules/explorer/_static/js/course.js]: @Sharonsyra Seen this? https://owlcarousel2.github.io/OwlCarousel2/docs/api-events.html

File: coursebuilder/views/base.html [hunk 3]
```diff
@@ -219,66 +219,57 @@
...
-         </a>
-
-         <a href="https://twitter.com/share?url=https://courses.academy.africa"
-            target="_blank" class="gbl-footer__sharing-control" aria-label="" share-width="" share-height="">
-             <img src="/modules/explorer/_static/images/twitter.png" alt="twitter" widht="25" height="22">
-         </a>
+          <div class="row">
+              <div class="col l8 m12 s12 footer-appstores">
+                  <div class="col s8">
+                  <a target="_blank" href="https://codeforafrica.org/">
```
Comment (chegejames) [inline coursebuilder/views/base.html]: @Sharonsyra indentation.

File: coursebuilder/modules/explorer/_static/js/course.js (hunk 2 above)
Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/js/course.js]: Thank you. Working on it.

**Comment by Sharonsyra** [comment]
@chegejames  Please check this PR. It has the same work only specific.

File: coursebuilder/modules/explorer/_static/js/course.js [hunk 4]
```diff
@@ -15,7 +15,9 @@ $(document).ready(function documentReady() {
   });
 
   $('.course-carousel').on('changed.owl.carousel', function(event) {
-    window.scrollTo(20, 20);
+    var el = $(".course-carousel");
+    var position = el.position();
```
Comment (chegejames) [inline coursebuilder/modules/explorer/_static/js/course.js]: @Sharonsyra are there cases where el.length > 1 ?

Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/js/course.js]: Yes. But in this case no since the div with the class name appears only once.

Comment (chegejames) [reply coursebuilder/modules/explorer/_static/js/course.js]: @Sharonsyra let us design for edge cases e.g https://courses.academy.africa/test/unit?unit=1&lesson=20

Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/js/course.js]: @chegejames I don't have access to it.

Comment (chegejames) [reply coursebuilder/modules/explorer/_static/js/course.js]: @Sharonsyra https://cfafrica-mooc-dev.appspot.com/test/unit?unit=4&lesson=11

Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/js/course.js]: Hey @chegejames still working on this. There is a bug. window.scrollTo() is always scrolling to the top even when the coordinates specify a different position in the page. Console.log() returns the correct position depending on the carousel clicked on though.

Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/js/course.js]: @chegejames this works perfectly now. You can test it out here. https://cfafrica-mooc-dev.appspot.com/test/unit?unit=4&lesson=13

File: coursebuilder/modules/explorer/_static/js/course.js [hunk 5]
```diff
@@ -16,7 +16,11 @@ $(document).ready(function documentReady() {
 
   $('.course-carousel').on('changed.owl.carousel', function(event) {
     var pos = $(this).position();
-    window.scrollTo(pos.top, pos.left);
+    window.scrollTo({
```
Comment (chegejames) [inline coursebuilder/modules/explorer/_static/js/course.js]: @Sharonsyra This fixed it?

Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/js/course.js]: Yes

File: coursebuilder/modules/explorer/_static/js/course.js (hunk 4 above)
Comment (chegejames) [reply coursebuilder/modules/explorer/_static/js/course.js]: @Sharonsyra do all browsers support / implement 'this'?

File: coursebuilder/modules/explorer/_static/js/course.js (hunk 5 above)
Comment (chegejames) [reply coursebuilder/modules/explorer/_static/js/course.js]: @Sharonsyra why?

Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/js/course.js]: I googled and the theory was that it was a browser issue but looks like it doesn't work on other browsers. It, however, works on chrome and safari though. Still working on this.

Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/js/course.js]: Hey Chege, check the event.target version here https://sharon-dot-cfafrica-mooc-dev.appspot.com/test/unit?unit=4&lesson=13

File: coursebuilder/modules/explorer/_static/js/course.js [hunk 6]
```diff
@@ -15,7 +15,8 @@ $(document).ready(function documentReady() {
   });
 
   $('.course-carousel').on('changed.owl.carousel', function(event) {
-    var pos = $(this).position();
+    var pos1 = jQuery(event.target)
```
Comment (chegejames) [inline coursebuilder/modules/explorer/_static/js/course.js]: @Sharonsyra Can we give this a more descriptive name? 

> There are only two hard things in Computer Science: cache invalidation and naming things'

Comment (chegejames) [reply coursebuilder/modules/explorer/_static/js/course.js]: @Sharonsyra also what does `$(event.target).closest('.course-carousel')` return?

Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/js/course.js]: @chegejames it works the same way. Achieves the same purpose. Curious could adding closest() be an overkill as by default event.target gets you the clicked carousel only.

Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/js/course.js]: I have also tested in on browserling and it works fine for opera, firefox, safari and chrome

Comment (chegejames) [reply coursebuilder/modules/explorer/_static/js/course.js]: @Sharonsyra Yes, it is overkill but it makes your intentions explicit as we may also want to get the position of the parent of the current .course-carousel and reposition to that.

Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/js/course.js]: @chegejames I see your point. It is done, please check it out now.
